go.sum

```diff
@@ -54,8 +54,8 @@ github.com/apparentlymart/go-textseg/v12 v12.0.0/go.mod h1:S/4uRK2UtaQttw1GenVJE
 github.com/apparentlymart/go-textseg/v13 v13.0.0 h1:Y+KvPE1NYz0xl601PVImeQfFyEy6iT90AvPUL1NNfNw=
 github.com/apparentlymart/go-textseg/v13 v13.0.0/go.mod h1:ZK2fH7c4NqDTLtiYLvIkEghdlcqw7yxLeM89kiTRPUo=
 github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5/go.mod h1:wHh0iHkYZB8zMSxRWpUBQtwG5a7fFgvEO+odwuTv2gs=
-github.com/auth0/go-auth0 v0.7.0 h1:mw7KbtVfAdVhJqinPeCNQQRuP4kv7eXe2GujaULgPWs=
```

Comment:
Should we use a feature branch for multi-PR work instead of leaving master in a non-releasable state?

Reply:
We won't be merging this until we update the go.mod to point to a correct stable release:) This is just to showcase that tests are passing with the work done in go-auth0.

Reply:
So, should the PR be a draft PR?

Reply:
Actually yes, that definitely makes more sense 👍🏻